docs/es/week12/12-2.md

```diff
@@ -0,0 +1,323 @@
+
+
+## Modelos de secuencia a secuencia
+
+
+### Modelos de lenguaje condicional
+
+Los modelos de lenguaje condicional no son útiles para generar muestras aleatorias de inglés, pero son útiles para generar un texto dado una entrada.
+
+Ejemplos:
+
+- Dada una oración en francés, generar la traducción al inglés
+- Dado un documento, generar un resumen
+- Dado un diálogo, generar la siguiente respuesta
+- Dada una pregunta, genera la respuesta
+
+
+### Modelos de secuencia a secuencia
+
+Generalmente, el texto de entrada está codificado. Esta incrustación resultante se conoce como un "vector de pensamiento", que luego se pasa al decodificador para generar tokens palabra por palabra.
+
+<centro>
+<img src = "{{site.baseurl}}/images/week12/12-2/s2s_Models.png" width = "60%" /> <br>
+<b> Fig. 3 </b>: Vector de pensamiento
+</center>
+
+
+### Transformador de secuencia a secuencia
+
+La variación de secuencia a secuencia de transformadores tiene 2 pilas:
+
+1. Pila de codificadores: la atención personal no está enmascarada, por lo que cada token en la entrada puede mirar todos los demás token en la entrada
+
+2. Pila de decodificadores: además de prestar atención a sí mismo, también presta atención a las entradas completas
+
+<centro>
+<img src = "{{site.baseurl}}/images/week12/12-2/s2s_transformers.png" width = "60%" /> <br>
+<b> Fig. 4 </b>: Transformador de secuencia a secuencia
+</center>
+
+Cada token en la salida tiene conexión directa con cada token anterior en la salida, y también con cada palabra en la entrada. Las conexiones hacen que los modelos sean muy expresivos y poderosos. Estos transformadores han realizado mejoras en la puntuación de traducción con respecto a los modelos recurrentes y convolucionales anteriores.
+
+
+## [traducción inversa] (https://www.youtube.com/watch?v=6D4EWKJgNn0&t=3811s)
+
+Al entrenar estos modelos, generalmente confiamos en grandes cantidades de texto etiquetado. Una buena fuente de datos son los procedimientos del Parlamento Europeo: el texto se traduce manualmente a diferentes idiomas que luego podemos utilizar como entradas y salidas del modelo.
+
+### Cuestiones
+
+- No todos los idiomas están representados en el parlamento europeo, lo que significa que no obtendremos pares de traducción para todos los idiomas en los que podríamos estar interesados. ¿Cómo encontramos texto para la capacitación en un idioma del que no necesariamente podemos obtener los datos?
+- Dado que los modelos como los transformadores funcionan mucho mejor con más datos, ¿cómo usamos el texto monolingüe de manera eficiente, *i.e.* sin pares de entrada/salida?
+
+Supongamos que queremos entrenar un modelo para que traduzca del alemán al inglés. La idea de la retrotraducción es entrenar primero un modelo inverso del inglés al alemán
+
+- Utilizando un bi-texto limitado podemos adquirir las mismas frases en 2 idiomas diferentes
```

Comment:
"bi-texto" is used here with a dash, but later it is used without it. For consistency, I'd suggest to always use it. Ditto for other words such as "retrotraducción"

Reply:
Good point!